src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -293,11 +293,10 @@ public void Track(ITelemetry telemetry)
             if (this.IsEnabled())
             {
                 this.Initialize(telemetry);
-                
-                this.WriteTelemetryToDebugOutput(telemetry);
```

Comment:
We should make a decision what direction we want to go here. I was under the impression we didn't want to show items that were filtered out by the processors. Do we always want to show them? Or should we only show them for sampling processor?